src/theme/web/theme.js

```diff
@@ -37,6 +37,13 @@ theme.color.greyLight = '#dbdbdb';
 theme.color.grey = '#aeaeae';
 theme.color.greyDark = '#727171';
 theme.color.greyDarker = '#4a4a4a';
+theme.color.jade = '#03B778'; // primary
+theme.color.jadeDark = '#108B65';
+theme.color.jadeLight = '#7ACAAE';
+theme.color.jadeLighter = '#E3F4DE';
+theme.color.lime = '#71C95B'; // trend
+theme.color.corn = '#FFD067'; // trip;
+theme.color.purple = '#BEABE9'; // tryst
```

Comment:
Remove the comments. As per design, you can always map the colors with the correct hex code.